### ISSUE TITLE
Show ReviewSubmission always on top of sections

### ISSUE THREAD

**Issue by nmadruga** [openmsupply/application-manager-web-app#563]
Can use Sticky for this - this might clash with SectionWrapper title also using Sticky so needs to be careful

**Comment by nmadruga** [comment]
@CarlosNZ & @andreievg I'm working on this... Currently implementing what is missing to get Option A - basically only styling (plus working on #581 on the same PR). Let me know if any of you think we should go ahead and implement Option B. Hopefully this would be helpful to have break-down before I jump in working with the Consolidation and style of Review page - or whoever jumps on those ones first : )

**Comment by andreievg** [comment]
Would option B just show pre-selected conform on non conform for level one reviewer who is not last level reviewer ?

For some reason I like option A more, maybe it's because of the above:
* Won't have anything to do apart from comment for lvl 1 reviewer (who is not last level)
* When commenting loosing context of what the comment is about

I like the hint though, that could be useful

**Comment by nmadruga** [comment]
Yes. I have implemented Option A, it's on a (PR)[664] now, I think that if we go with Option B, we need more additions to it, for example to solve problem:
> * When commenting loosing context of what the comment is about

I suggest having the LOQ displaying at this Submission page as well.


> Would option B just show pre-selected conform on non conform for level one reviewer who is not last level reviewer ?

Yes, I think either pre-selected (which is already implemented - but showing on the bottom of the page in Option A) or not take the user to the Submission page in case no `last-level-review`, we could simply show a modal with the hint saying something like "You have submitted review to supervisor. Thanks" : )
